toshi-magento-plugin/Shipping/Helper/Data.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace Toshi\Shipping\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\HTTP\Client\Curl;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+
+class Data extends AbstractHelper
+{
+    /**
+     * @var Curl
+     */
+    protected $curlClient;
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    const MODE = 'carriers/toshi/mode';
+    const ENVIRONMENT = 'carriers/toshi/environment';
+    const CLIENT_API_KEY = 'carriers/toshi/toshi_client_api_key';
+    const SERVER_API_KEY = 'carriers/toshi/toshi_server_api_key';
+    const ENDPOINT_URL = 'carriers/toshi/toshi_endpoint_url';
+    const MIN_BASKET_AMOUNT = 'carriers/toshi/toshi_min_basket_amount';
+    const SIZE_ATTRIBUTE = 'carriers/toshi/size_attribute';
+    const COLOR_ATTRIBUTE = 'carriers/toshi/color_attribute';
+    const TIMEOUT = 'carriers/toshi/timeout';
+    const HOLIDAYS = 'carriers/toshi/holidays';
+    const DEFERRED_DAYS = 'carriers/toshi/deferred_days';
+    const TAX_POSTCODE = 'tax/defaults/postcode';
+
+    const ADDRESS_ELIGIBLE_ENDPOINT = '/api/v2/address/eligible';
```

Comment:
We've updated our versioning of our api. This will now need to be /v2/address/eligible. 

The root domain has also been changed to api.toshi.co

Reply:
I can make that change to the the const here. 
In regards to the root domain, will you still be having different root domains for api.toshi.co and a staging version of that?

Reply:
Hi Minesh 

Yes you're right prod is now api.toshi.co and staging is staging.api.toshi.co

Reply:
Hey James, 
The Domains shouldn't be a problem as they are defined within the configuration, the URL change can be done pretty quickly for the endpoints.